pkg/restore/import.go

```diff
@@ -233,6 +250,19 @@ func (importer *FileImporter) downloadSST(
 		return nil, true, errRewriteRuleNotFound
 	}
 	sstMeta := getSSTMetaFromFile(id, file, regionInfo.Region, regionRule)
+	// For raw kv mode, cut the SST file's range to fit in the restoring range.
+	if importer.isRawKvMode {
+		if bytes.Compare(importer.rawStartKey, sstMeta.Range.GetStart()) > 0 {
```

Comment:
Why not use the range of the region?

Reply:
It's possible that we are restoring a range that's a subrange of backup-ed data, and we need to filter out keys that are not in the range, and the range's border may be neither region's border nor SST file's border. So cut the range of SSTMeta here.

Reply:
But we download and ingest SSTs per region, use the range of backup data may cause a region ingests extra data.

Reply:
No, I only meant to cut the range but never enlarge it. So the only possibility is that the range is made smaller than before. Is my code wrong?